CMakeLists.txt

```diff
@@ -38,15 +43,23 @@ if(BUILD_GAME)
 	set(GAME_SRC ${GAME_SRC} ${SHARED_SRC} ${LIBRARY_SRC})
 
 	link_directories("Shared")
-	link_directories("Libraries/sdl/x86")
-
-	add_executable(game WIN32 ${GAME_SRC})
-	set_target_properties(game PROPERTIES OUTPUT_NAME game)
-	set_target_properties(game PROPERTIES LINKER_LANGUAGE CXX)
-
-	set(SDL_LIBRARY SDL2.lib SDL2main.lib SDL2test.lib SDL2_net.lib)
 
-	target_link_libraries(game ${STATIC_LIBRARIES} ${SDL_LIBRARY})
+	if(WIN32)
+		link_directories("Libraries/sdl/x86")
+		add_executable(game WIN32 ${GAME_SRC})
```

Comment:
why duplicate the stuff in common ? just make sure  ${SDL_LIBRARY} and  ${STATIC_LIBRARIES} are set correctly for all the platforms or just put the target_link_libraries in the if else